service/core.go

```diff
@@ -126,25 +126,25 @@ func (s *serviceCore) AuthClient() auth.Client {
 }
 
 func (s *serviceCore) Name() string {
```

Comment:
I think Name is fine. The main reason I changed the names is so it's clear what's missing in the error messages :) when you have a service object, calling Name() makes more sense since it is the service name and id can indicate a number.